post-scan-actions/azure-python-promote-or-quarantine/README.md

```diff
@@ -0,0 +1,74 @@
+# Cloud One File Storage Security Post Scan Action for Azure Storage Account - Promote or Quarantine
+
+After a scan occurs, this example Azure function places clean files in one storage account and malicious files in another.
+
+## Prerequisites
+
+1. **Install supporting tools**
+    - Install the Azure command line interface (CLI). See [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) for details.
+    - Install the Azure Functions Core Tools. See [Work with Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local) for details.
+1. **Create Azure storage accounts and connection strings**
+    - Create a 'Promote storage account' to receive clean files. Example: `fsspromote`.
+    - In the Azure portal, go to **Storage account** > Promote storage account > **Access keys**, click **Show keys**, and copy the value of **Connection string** of either key.
+    - Create a 'Quarantine storage account' to receive quarantined files. Example: `fssquarantine`.
+    - In the Azure portal, go to **Storage account** > Quarantine storage account > **Access keys**, click **Show keys**, and copy the value of **Connection string** of either key.
+1. **Find the scanning blob storage account resource ID and the service bus topic resource ID**
+    - In the Azure portal, go to **Resource group** > your storage stack > **Deployments > storageStack > Outputs**.
+    - Copy the value of **blobStorageAccountResourceID** to a temporary location. Example: `/subscriptions/12345678-1111-1111-1111-123456789012/resourceGroups/my-storages/providers/Microsoft.Storage/storageAccounts/scanningstorage`.
+    - Copy the value of **scanResultTopicResourceID** to a temporary location. Example: `/subscriptions/12345678-1111-1111-1111-123456789012/resourceGroups/my-storage-stack/providers/Microsoft.ServiceBus/namespaces/tmsrt0abcd56789abcd/topics/scan-result-topic`.
+
+## Installation
+
+### 1. Deploy Azure function app
+
+1. Login Azure CLI and switch to the Azure subscription to deploy resources:
```

Comment:
1. Log in to the Azure CLI and switch to the Azure subscription to deploy resources: